Assets/MRTK/SDK/Experimental/StepSlider/StepSlider.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI
+{
+    /// <summary>
+    /// A slider with a fixed number of step values that can be moved by grabbing / pinching a slider thumb
+    /// Number of steps defaults to 0, if number isnt set then slider behaves just like a regular PinchSlider
+    /// </summary>
+    [AddComponentMenu("MRTK/SDK/Experimental/StepSlider")]
+    public class StepSlider : PinchSlider, IMixedRealityPointerHandler
+    {
+
+        #region Serialized Fields and Properties
+        [Experimental]
+        [SerializeField]
+        [Tooltip("Number of subdvisions the slider is split into.")]
+        
+        private int sliderStepDivisions = 0;
+        /// <summary>
+        /// Property accessor of sliderStepDivisions, it holds the number of subdvisions the slider is split into.
+        /// </summary>
+        public int SliderStepDivisions
+        {
+            get 
+            { 
+                return sliderStepDivisions; 
+            }
+            set
+            {
+                sliderStepDivisions = value;
+            }
+        }
+        #endregion
+
+        #region Protected Members
+        /// <summary>
+        /// Private member used to adjust slider values
+        /// </summary>
+        protected float sliderStepVal = 0.01f;
+        #endregion
+
+        #region Unity methods
+        new void Start()
```

Comment:
For clarity and future extensibility, could you make any method you needed to `new` here `protected virtual` in the underlying `PinchSlider` implementation? Then these could be `protected override` instead of `new`. Also helps with matching the [access modifier guidance](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers).

Reply:
Since this has been open for a while, I didn't want to block on my late review, so I just pushed the changes.